### PR TITLE
Show LNURL-pay responses in the app instead of a notification

### LINKED ISSUE #342: LNURL response notification is not shown in chrome
In Firefox the LNURL response message (e.g. thanks for the payment) is shown as notification.
In chrome I only see the payment success notification with the pre-image. 

Maybe this is a bug, maybe this is only how my chrome notifications are configured?

### PR DESCRIPTION
**Pull request by dylancom**
Fixes #342

### CODE REVIEW

File: src/app/screens/LNURLPay.tsx [hunk 1]
```diff
@@ -110,7 +110,9 @@ function LNURLPay(props: Props) {
         }
       }
 
-      window.close();
+      if (shouldCloseWindow) {
+        window.close();
```
Comment (bumi) [inline src/app/screens/LNURLPay.tsx:114]: if we move the window.close() into the switch case, then we do not need to set a new variable.
and it is not much difference between writing: `shouldCloseWindow = true/false` and `window.close();`

what do you think?

**Review by bumi — COMMENTED** [review]
we also could do a simple `alert` :) but I guess this is much nicer (haven't tested it yet)

File: src/app/screens/LNURLPay.tsx (hunk 1 above)
Comment (dylancom) [reply src/app/screens/LNURLPay.tsx:114]: What if the code doesn't pass the `(successAction && !payment.payment_error)` condition? Currently we'd close the window then too.

Comment (bumi) [reply src/app/screens/LNURLPay.tsx:114]: 👍

**Comment by bumi** [comment]
two thoughts: 

while we are at that: shall we also convert the alert in the `url` tag to display such a message with a link (that opens in a new tab)? 

and we might need a `close` link? - when I tested it I had the question how to close/get back to the home screen. ?

**Comment by dylancom** [comment]
I added a close button, which indeed was missing.
For the alert, I'm not sure to open a new tab as the spec speaks about a popup.

**Comment by bumi** [comment]
> For the alert, I'm not sure to open a new tab as the spec speaks about a popup.

I think opening a tab is the easiest (as we do right now) and the ones I have seen so far have been full web pages.

File: src/app/screens/LNURLPay.tsx [hunk 2]
```diff
@@ -212,42 +205,96 @@ function LNURLPay(props: Props) {
     return elements;
   }
 
+  function renderSuccessAction() {
+    if (successAction) {
```
Comment (bumi) [inline src/app/screens/LNURLPay.tsx:209]: maybe `if (!successAction) { return; }` to prevent the nesting?

**Review by bumi — APPROVED** [review]

File: src/app/screens/LNURLPay.tsx (hunk 2 above)
Comment (dylancom) [reply src/app/screens/LNURLPay.tsx:209]: Will remove the check and make TS assume the value is present.
As renderSuccessAction is only called when successAction exists.

**Pull request merged**